docs/en/user_guides/4_train_test.md

```diff
@@ -1,110 +1,85 @@
 
-```python
-# use the pre-trained model for the whole PSPNet
-load_from = 'https://download.openmmlab.com/mmsegmentation/v0.5/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth'  # model path can be found in model zoo
-```
+## Training and testing on a single GPU
 
 ### Training on a single GPU
 
 We provide `tools/train.py` to launch training jobs on a single GPU.
 The basic usage is as follows.
 
 ```shell
-python tools/train.py \
-    ${CONFIG_FILE} \
-    [optional arguments]
+python tools/train.py  ${CONFIG_FILE} [optional arguments]
 ```
 
 This tool accepts several optional arguments, including:
 
 - `--work-dir ${WORK_DIR}`: Override the working directory.
 - `--amp`: Use auto mixed precision training.
-- `--resume ${CHECKPOINT_FILE}`: Resume from a previous checkpoint file. If not specify, try to auto resume from the latest checkpoint in the work directory.
-- `--cfg-options ${OVERRIDE_CONFIGS}`: Override some settings in the used config, the key-value pair in xxx=yyy format will be merged into config file.
+- `--resume`: Resume from the latest checkpoint in the work_dir automatically.
+- `--cfg-options ${OVERRIDE_CONFIGS}`: Override some settings in the used config, and the key-value pair in xxx=yyy format will be merged into the config file.
   For example, '--cfg-option model.encoder.in_channels=6'. Please see this [guide](./1_config.md#Modify-config-through-script-arguments) for more details.
-  Below is the optional arguments for multi-gpu test:
+
+Below are the optional arguments for the multi-gpu test:
+
 - `--launcher`: Items for distributed job initialization launcher. Allowed choices are `none`, `pytorch`, `slurm`, `mpi`. Especially, if set to none, it will test in a non-distributed mode.
 - `--local_rank`: ID for local rank. If not specified, it will be set to 0.
 
-**Note**:
-Difference between `--resume` and `load-from`:
-`--resume` loads both the model weights and optimizer status, and the iteration is also inherited from the specified checkpoint.
-It is usually used for resuming the training process that is interrupted accidentally.
+**Note:** Difference between the argument `--resume` and the field `load-from` in the config file:
 
-`load-from` only loads the model weights and the training iteration starts from 0. It is usually used for fine-tuning.
+`--resume` only determines whether to resume from the latest checkpoint in the work_dir. It is usually used for resuming the training process that is interrupted accidentally.
 
-### Training on CPU
+`load-from` will specify the checkpoint to be loaded and the training iteration starts from 0. It is usually used for fine-tuning.
 
-The process of training on the CPU is consistent with single GPU training if machine does not have GPU. If it has GPUs but not wanting to use it, we just need to disable GPUs before the training process.
```

Comment:
**Note**:
If you would like to resume training from a specific checkpoint, you can use `--resume` with `--cfg-options load-from=$CHECKPOINT`.